frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -66,38 +66,47 @@ const ModelDetailsPage: React.FC = conditionalArea(
     [modelCatalogSources, decodedParams],
   );
 
-  const registerModelButton = (isSecondary = false) =>
```

Comment:
You don't want to get rid of the `isSecondary` argument here. It was making this register button be secondary in the details page header but primary in the ilab info popover, we still need that behavior.